package.json

```diff
@@ -68,6 +68,7 @@
     "@typescript-eslint/eslint-plugin": "^4.4.0",
     "@typescript-eslint/parser": "^4.4.0",
     "babel-eslint": "^10.1.0",
+    "babel-jest": "^26.5.2",
```

Comment:
Was this a peer dependency?

Reply:
If I recall correctly, yes, this was added to mitigate one of those peer dependency Yarn warnings.